tests/network/vmi_slirp_interface.go

```diff
@@ -207,17 +205,11 @@ var _ = SIGDescribe("Slirp Networking", decorators.Networking, func() {
 			setDefaultNetworkInterface("bridge")
 		})
 		It("should reject VMIs with default interface slirp when it's not permitted", func() {
-			var t int64 = 0
-			vmi := tests.NewRandomVMI()
-			vmi.Spec.TerminationGracePeriodSeconds = &t
-			// Reset memory, devices and networks
-			vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("128Mi")
-			vmi.Spec.Domain.Devices = v1.Devices{}
-			vmi.Spec.Networks = nil
-			tests.AddEphemeralDisk(vmi, "disk0", v1.DiskBusVirtio, cd.ContainerDiskFor(cd.ContainerDiskCirros))
+			vmi := libvmi.NewGuestless()
 
 			vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), vmi)
 			Expect(err).To(HaveOccurred())
+			Expect(err.Error()).To(ContainSubstring("Slirp interface is not enabled in kubevirt-config"))
```

Comment:
```
It("should reject VMIs with default interface slirp when it's not permitted", func() {
			_, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), libvmi.NewGuestless())
			Expect(err).To(HaveOccurred())
			Expect(err.Error()).To(ContainSubstring("Slirp interface is not enabled in kubevirt-config"))
		})
```

Reply:
I don't understand your comment. What are you suggesting?

Reply:
Instead of having empty line and unused `vmi` (`err` is only used):
```
It("should reject VMIs with default interface slirp when it's not permitted", func() {
	vmi := libvmi.NewGuestless()

	vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), vmi, metav1.CreateOptions{})
	Expect(err).To(HaveOccurred())
	Expect(err.Error()).To(ContainSubstring("Slirp interface is not enabled in kubevirt-config"))
})
```
I suggested to have it shorter and remove unused variable:
```
It("should reject VMIs with default interface slirp when it's not permitted", func() {
	_, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), libvmi.NewGuestless(), metav1.CreateOptions{})
	Expect(err).To(HaveOccurred())
	Expect(err.Error()).To(ContainSubstring("Slirp interface is not enabled in kubevirt-config"))
})
```